docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+web:
+  build: .
+  command: 'bash -c ''npm start'''
```

Comment:
Good point. Changed in most recent commit https://github.com/ftlabs/screens/commit/81bb2fb692d09e3a39b25a2c216f8bc9c22b268c